### DISCUSSION TITLE
How to specify path to Node exe?

### DISCUSSION THREAD

**Discussion by krista-prokop** [cypress-io/cypress#29541] [Questions and Help]
I tried setting a custom node path in cypress-config.ts but it doesn't do anything. There's no docs on this and i'm wondering what I'm missing.   My company's CI servers now enforce node v20+ so our cypress tests now fail.

Added path to exe:
```
export default defineConfig({
	// resolvedNodeVersion: '20.9.0',
	resolvedNodePath: '../path/to/node/node.exe'      //  also tried `../path/to/node/`
})
```

The output is the same as always though:

```
(Run Starting)  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ Cypress:        13.9.0                                                                         │
  │ Browser:        Electron 118 (headless)                                                        │
  │ Node Version:   v18.20.2 (C:\Program Files\nodejs\node.exe)                                        │
  │ Specs:          7 found (...)                                                     │
  │ Searched:       cypress/test/e2e/**/*.cy.{js,jsx,ts,tsx}                                       │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘

```

**Comment by portyu9** [comment]
Hi @krista-prokop 

`resolvedNodePath` is not a supported configuration option for selecting which Node executable Cypress should run with. It represents the resolved Node runtime.

For Cypress 13.x, Cypress uses the Node.js runtime it was launched with. Configure your CI environment so the required Node installation is active **before** starting Cypress.

On Windows, check:

```bat
where node
node --version
where npx
```

Then run Cypress from that same environment. If those commands still point to the old `C:\Program Files\nodejs\node.exe`, fix the CI `PATH`/Node setup rather than setting `resolvedNodePath` in `cypress.config.ts`.

For the version in this thread (`13.9.0`), Node 20 was appropriate. If upgrading today, note that Cypress 16 requires Node 22.x, 24.x, or 26+.

Please mark this answer as accepted if it helped, thank you.
